### DISCUSSION TITLE
How to force Github to update my repository files quickly?

### DISCUSSION THREAD

**Discussion by R3uan3** [community/community#34000] [Other Feature Feedback, Questions, & Ideas]
Scenario:
I have file `test.txt` on my repo containing data `"abc"` when I modify the content of the text file to `"xyz"` and download it using `curl` it takes like 10min for the file being downloaded to be updated to `xyz`

How to 'force' GitHub to update the edited file quickly? do I need a 'paid' account for such a thing?

**Comment by Blackpearlcodes** [comment]
git add .
git commit -m "my changes" 
git remote add origin https://www.somewebsite.com
git push -u origin master

**Reply by R3uan3** [reply]
The file is updated in the repository however, when I download it contains outdated data, takes a lot of time to download the updated the new data.

**Reply by Blackpearlcodes** [reply]
it shouldn't take time if you are doing everything correctly, make a clone of your repo to your local computer and it should reflect the changes directly. No need for a paid account.

**Comment by pavan2108** [comment]
It happend same with me while ago I fixed the issue by removing the cached data stored in our github repositories
```
git rm -r --cached .
git add .
```
Credits : [https://stackoverflow.com/questions/1274057/how-do-i-make-git-forget-about-a-file-that-was-tracked-but-is-now-in-gitignore](https://stackoverflow.com/questions/1274057/how-do-i-make-git-forget-about-a-file-that-was-tracked-but-is-now-in-gitignore)
